cisco_aci/assets/dashboards/cisco_aci_dashboard.json

```diff
@@ -3,559 +3,1065 @@
+                                "sum"
+                            ],
+                            "type": "timeseries",
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "formula": "query1 + query2"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "sum:cisco_aci.fabric.port.fault_counter.warn{$pod} by {node_id,fabric_pod_id}"
+                                        },
+                                        {
+                                            "data_source": "metrics",
+                                            "name": "query2",
+                                            "query": "sum:cisco_aci.fabric.port.fault_counter.crit{$pod} by {node_id,fabric_pod_id}"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "style": {
+                                        "palette": "purple",
+                                        "line_type": "solid",
+                                        "line_width": "normal"
+                                    },
+                                    "display_type": "bars"
+                                }
+                            ],
+                            "custom_links": []
+                        },
+                        "layout": {
+                            "x": 6,
+                            "y": 0,
+                            "width": 6,
+                            "height": 2
                         }
-                    }
-                ],
-                "custom_links": [],
-                "title": "Top 10 EPGs by traffic",
-                "title_size": "16",
-                "title_align": "left",
-                "time": {
-                    "live_span": "1h"
-                }
-            },
-            "layout": {
-                "x": 57,
-                "y": 64,
-                "width": 38,
-                "height": 21
-            }
-        },
-        {
-            "id": 18,
-            "definition": {
-                "type": "query_value",
-                "requests": [
+                    },
                     {
-                        "q": "count_not_null(avg:cisco_aci.tenant.ingress_pkts.flood.cum{*} by {application})",
-                        "aggregator": "last"
+                        "id": 10,
+                        "definition": {
+                            "title": "Bandwidth by Switch (ingress is negative)",
```

Comment:
Any reason to display the ingress as negative?

Reply:
just a relic of the old dashboard (
![image](https://github.com/DataDog/integrations-core/assets/9274242/a7f43f66-0b35-4eb1-9863-697d2724b86c)
)
but i can change it such that they're separate, it's probably easier to parse tbh